src/o365/yammer/commands/group/group-user-remove.ts

```diff
@@ -0,0 +1,150 @@
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import YammerCommand from "../../../base/YammerCommand";
+import request from '../../../../request';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: number;
+  userId?: number;
+  confirm?: boolean;
+}
+
+class YammerGroupUserRemoveCommand extends YammerCommand {
+  constructor() {
+    super();
+  }
+
+  public get name(): string {
+    return `${commands.YAMMER_GROUP_USER_REMOVE}`;
+  }
+
+  public get description(): string {
+    return 'Removes a user from a Yammer group';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.id = args.options.id !== undefined;
```

Comment:
Since `id` is required, there is no need to track it explicitly in the telemetry because it would always show up as filled in